dataviz/dataviz-in-practice.qmd

```diff
@@ -534,7 +534,7 @@ p <- gapminder |>
   filter(year %in% years & country %in% country_list) |>
   mutate(group = ifelse(group == "West", "West", "Developing")) |>
   ggplot(aes(dollars_per_day, y = after_stat(count), fill = group)) +
-  scale_x_continuous(trans = "log2", limits = c(0.125, 300))
+  scale_x_continuous(transform = "log2", limits = c(0.125, 300))
 p + geom_density(alpha = 0.2) + facet_grid(year ~ .)
 ```
 
```

Comment:
I think a new heading is warranted somewhere around here, as we are no longer talking about computed aesthetics.